app/controllers/bz_controller.rb

```diff
@@ -981,43 +990,51 @@ def linked_in_export_guts
               end
 
               linkedin_data.linkedin_id = item["id"] = info["id"]
-              linkedin_data.first_name = item["first-name"] = info["firstName"]
-              linkedin_data.last_name = item["last-name"] = info["lastName"]
-              linkedin_data.maiden_name = item["maiden-name"] = info["maidenName"]
-              linkedin_data.email_address = item["email-address"] = info["emailAddress"]
-              linkedin_data.location = item["location"] = info["location"]["name"] unless info["location"].nil?
-              linkedin_data.industry = item["industry"] = info["industry"]
-              linkedin_data.job_title = item["job-title"] = get_job_title(info["threeCurrentPositions"])
-              linkedin_data.num_connections = item["num-connections"] = info["numConnections"]
-              linkedin_data.num_connections_capped = item["num-connections-capped"] = info["numConnectionsCapped"]
-              linkedin_data.summary = item["summary"] = info["summary"]
-              linkedin_data.specialties = item["specialties"] = info["specialties"]
-              linkedin_data.public_profile_url = item["public-profile-url"] = info["publicProfileUrl"]
+              linkedin_data.first_name = item["first-name"] = info["localizedFirstName"]
+              linkedin_data.last_name = item["last-name"] = info["localizedLastName"]
+              linkedin_data.maiden_name = item["maiden-name"] = info["localizedMaidenName"]
+              # no longer available
+              # linkedin_data.email_address = item["email-address"] = info["emailAddress"]
+              linkedin_data.location = item["location"] = info["location"]["postalCode"] unless info["location"].nil?
+              linkedin_data.industry = item["industry"] = info["industryName"]["localized"]["en_US"]
+
+              linkedin_data.job_title = item["job-title"] = get_job_title(info["positions"])
+              linkedin_data.num_connections = item["num-connections"] = nil # removed in V2
```

Comment:
Can you queue up a task to document and train the Data team on which fields we have historical data for but which are obsolete as of June 30th, 2019? We prob want to remove the cols from the DB, so I'd like to have to task remind us to have that convo too.